### PR TITLE
Add ended indicator in vaults

### LINKED ISSUE #322: Add Ended indicator after the vault has ended
Same design like this for ended: 
image

### PR DESCRIPTION
**Pull request by medardm**
closes #322

### CODE REVIEW

**Review by swengr-janan — APPROVED** [review]

File: src/utils/farmHelpers.ts [hunk 1]
```diff
@@ -33,6 +34,17 @@ export const calculateUserRewardRate = (
     ?.toSignificant(4)
 }
 
+export const getFarmStatus = (farm: Farm) => {
```
Comment (kojiadrianojr) [inline src/utils/farmHelpers.ts:37]: Question ko lang kuys ini. Hindi ba mas ideal na su sarong info lang kang Vault ang ipasa sa function na ini? para dae dakulang load. boolean lang naman sana need dito. 

What ya think?

File: src/utils/poolHelpers.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { Pool } from '../state/types'
+
+const getPoolStatus = (pool: Pool) => {
```
Comment (kojiadrianojr) [inline src/utils/poolHelpers.ts:3]: Same here, question lang. Di kaya boolean lang naman need why need ipasa yung buong vaults info sa function na ini.

**Comment by kojiadrianojr** [comment]
Pwede rin ideconstruct if ung info kang pool/farm like coming soon para yun lang ipasa. 

example:
`const { comingSoon } = farms/pools`

**Comment by kojiadrianojr** [comment]
Ay nevermind my comments kuys here. naiisip ko kasi ended and comingSoon su mean ko pero if mas okay sainyo ipasa buong farms/pools instead na as object nalang na {ended, comingSoon} okay lang naman siguro both.

**Pull request merged**